### PR TITLE
LPS-143420

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina,
this is a redone of the first task of [LPS-143403](https://issues.liferay.com/browse/LPS-143403)
Process in this can be found here: [sheet](https://docs.google.com/spreadsheets/d/1WDPW9MQANZCKVUvqKoBtbj1cIo4S-IsUS7QNDWJp9VA/edit#gid=412471923)

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by tinatian** [comment]
@julschong
Can you make changes like this? 
For each ***util, you did:
1. remove the usages in modules
2. Modify util-spring to expose corresponding OSGi service if needed
3. Modify SF to make sure you cleaned all the usages in modules
This way it would be clear what you did for each ***Util.

Thanks.
Tina.

File: portal-impl/portal-impl.iml [hunk 1]
```diff
@@ -1,68 +1,73 @@
...
-		<orderEntry type="module" module-name="util-java" />
-		<orderEntry type="module" module-name="util-taglib" />
-		<orderEntry type="module" module-name="portal-kernel" scope="PROVIDED" />
-		<orderEntry type="module" module-name="portal-test" scope="TEST" />
-		<orderEntry type="module" module-name="portal-test-integration" scope="TEST" />
-		<orderEntry type="library" scope="PROVIDED" name="development" level="project" />
-		<orderEntry type="library" name="portal" level="project" />
-		<orderEntry type="library" scope="PROVIDED" name="global" level="project" />
-	</component>
+  <component name="FacetManager">
```
Comment (tinatian) [inline portal-impl/portal-impl.iml:3]: @julschong This portal-impl.iml seems unrelated to other changes in the same commit.

Tina.
